src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -2456,16 +2496,16 @@ private bool HasVariantConversion(NamedTypeSymbol source, NamedTypeSymbol destin
                 return quickResult.Value();
             }
 
-            return this.CreateInstance(_currentRecursionDepth + 1).
+            return this.CreateInstance(_currentRecursionDepth + 1, IncludeNullability).
```

Comment:
>IncludeNullability [](start = 67, length = 18)

Would it make sense to inherit this state by default?

Reply:
Yes, that makes sense. Done.

---
In reply to: [168041425](https://github.com/dotnet/roslyn/pull/23981#discussion_r168041425) [](ancestors = 168041425)